ecs-init/docker/docker_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build test
```

Comment:
these tests were not running because of these flags, so some had to be fixed as they havent been passing for a long time (or possibly not ever)